src/Organisms/InProgressContent/InProgressContent.js

```diff
@@ -0,0 +1,25 @@
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import FakeButton from '../../Atoms/FakeButton/FakeButton';
+import InProgressContentWrapperStyled from './InProgressContentWrapperStyled';
+import inprogress from '../../Assets/inprogress.png';
+
+const InProgressContent = () => {
+  return (
+    <ColumnWrapper paddingLeftRight={1}>
+      <InProgressContentWrapperStyled>
+        <img src={inprogress} alt="in progress" />
+        <div>
+          <h3>Strona w trakcie przygotowania!</h3>
+          <p>Strona dostępna wkrótce!</p>
+        </div>
+        <FakeButton
+          secondary="secondary"
+          text="Powrót do strony głównej"
```

Comment:
I know it is in Figma, but maybe better if it will send user to previous page?

Reply:
@pierwszazlewej co Ty na to?